DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/onchip_memory_cache/sample.json

```diff
@@ -5,9 +5,18 @@
   "description": "Intel® FPGA tutorial demonstrating the caching of on-chip memory to reduce loop initiation interval",
   "toolchain": ["dpcpp"],
   "os": ["linux", "windows"],
-  "builder": ["ide", "cmake"],
   "targetDevice": ["FPGA"],
+  "builder": ["ide", "cmake"],
```

Comment:
I'm not sure if it makes a difference, but you moved the builder entry one line down.
It should not make a difference, but to avoid braking a flow somewhere, we might as well let it where it was before

Reply:
I did this to match the sample.json I was using as a reference (pipe_array), which I know passes the CI.